providers/provider-pojo/src/main/java/io/servicecomb/provider/pojo/Invoker.java

```diff
@@ -86,20 +87,48 @@ protected void prepare() {
   @Override
   public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
     if (swaggerConsumer == null) {
-      prepare();
+      synchronized (this) {
+        if (swaggerConsumer == null) {
```

Comment:
swaggerConsumer should be volatile.

Reply:
done

Reply:
@wujimin  did you submit the change, I cannot find the change.

Reply:
maybe you did not refresh the page?